### PR TITLE
[INLONG-5776][Manager] Add tenant param to InlongGroup that uses Pulsar

### LINKED ISSUE #5776: [Improve][Manager] Add tenant param for Pulsar info
### Description

Add tenant param for Pulsar info.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Prepare a Pull Request

- Fixes #5776

### Motivation

**Add tenant param to InlongGroup that uses Pulsar.**

Currently, the tenant parameter is stored in the Pulsar cluster information, which is not reasonable.

Because a Pulsar cluster supports many tenants, just like its namespaces and topics, **it is a one-to-many relationship with the cluster, not a one-to-one relationship.**

### Modifications

1. Add `tenant` param for `InlongPulsarInfo`.
2. First, get the tenant from `InlongPulsarInfo`, and then get it from the `PulsarCluster`, finally, use the default tenant `public`.

### Verifying this change

- [x] This change is a trivial rework/code cleanup without any test coverage.

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Comment by gong** [comment]
@chestnut-c please help to review it

**Review by chestnut-c — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Comment by vernedeng** [comment]
This is a reasonable change, but it may lead to a configure failure for Sort-Standalone and DataProxy.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/consume/apply/ApproveConsumeProcessListener.java [hunk 1]
```diff
@@ -130,17 +130,17 @@ private void createPulsarSubscription(InlongConsumeEntity entity) {
         ClusterInfo clusterInfo = clusterService.getOne(clusterTag, null, ClusterType.PULSAR);
         PulsarClusterInfo pulsarCluster = (PulsarClusterInfo) clusterInfo;
         try (PulsarAdmin pulsarAdmin = PulsarUtils.getPulsarAdmin(pulsarCluster)) {
-            PulsarTopicInfo topicMessage = new PulsarTopicInfo();
-            String tenant = pulsarCluster.getTenant();
+            InlongPulsarDTO pulsarDTO = InlongPulsarDTO.getFromJson(groupEntity.getExtParams());
+            String tenant = pulsarDTO.getTenant();
             if (StringUtils.isEmpty(tenant)) {
```
Comment (gosonzhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/consume/apply/ApproveConsumeProcessListener.java:135]: isEmpty -- > isBlank

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java [hunk 2]
```diff
@@ -111,12 +112,25 @@ public InlongConsumeInfo getFromEntity(InlongConsumeEntity entity) {
         }
         String groupId = entity.getInlongGroupId();
         InlongGroupInfo groupInfo = groupService.get(groupId);
+        if (!(groupInfo instanceof InlongPulsarInfo)) {
```
Comment (gosonzhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java:115]: Why not use mqType to convert directly?

**Comment by healchow** [comment]
> This is a reasonable change, but it may lead to a configure failure for current SortStandalone and DataProxy. There are two important things to adapt this modification:
> 
> 1. A update manuscript from last version to the newest one,
> 2. SortSandalone and DataProxy should change the way to acquire "tenant".

Thanks for your review🎉

As I considered compatibility with DataProxy and SortStandalone, **I did not add the default value for the tenant in the InlongPulsarRequest**, see the code:

https://github.com/apache/inlong/blob/dea5e1fb9d9f9dad572b7a76e9e4e136b6376779/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/group/pulsar/InlongPulsarRequest.java#L39-L44

That is to say, if you do not set the tenant value when creating or modifying InlongGroup, DataProxy and SortStandalone will read the tenant from the PulsarCluster when they obtain the tenant, because there is no tenant in InlongGroup.

This way should be compatible with historical data. And finally, if you want to upgrade all historical data, you need to modify the request methods of DataProxy / SortStandalone and copy the tenant configuration from PulsarClusters to the related InlongGroups.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/consume/apply/ApproveConsumeProcessListener.java (hunk 1 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/consume/apply/ApproveConsumeProcessListener.java:135]: Modified.

**Review by vernedeng — APPROVED** [review]
lgtm

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java (hunk 2 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java:115]: It's just a double-check, which seems unnecessary and has been removed.

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**